server/src/main/java/io/unitycatalog/server/service/credential/aws/AwsCredentialVendor.java

```diff
@@ -19,10 +19,10 @@ public AwsCredentialVendor(ServerProperties serverProperties) {
 
   private CredentialsGenerator createCredentialsGenerator(S3StorageConfig config) {
     // Dynamically load and initialize the generator if it's intentionally configured.
-    if (config.getCredentialGenerator() != null) {
+    if (config.getCredentialsGenerator() != null) {
```

Comment:
Add the `s` to match the server property name, please see: https://github.com/unitycatalog/unitycatalog/blob/main/server/src/main/java/io/unitycatalog/server/utils/ServerProperties.java#L101

Reply:
actually.. if we have to be grammatically correct ... it should "credentialGenerator" 
its like, we say "fruit market", not "fruits market" 

so better to fix the property :)

Reply:
Thanks for the comment,  have the same question before.  I choose this because the aws credentials provider use the `AwsCredentialsProvider` name. Please see https://github.com/unitycatalog/unitycatalog/blob/main/connectors/spark/src/main/scala/io/unitycatalog/spark/auth/AwsVendedTokenProvider.java#L7

That's why I change to `credentialsGenerator`.  For the aws name,  seems like it's a provider which will generate many new credentials. that's why they add a `s`. 

I don't have strong opinion about this, if you prefer the `credentialGenerator`.  I can change it.

Reply:
The point for me is: we may need a consistent naming.

Reply:
100% agree on consistent naming. 
since there is precedence on AwsCredentialsProvider, lets go with `credentialsGenerator` 👍